csphotoselector.css

```diff
@@ -278,3 +278,41 @@
 .CSPhoto_container_active {
   left: 11px;
 }
+
+/* responsive support */
+/* optimized for iPhone 5 and upper
+ * will produce a 1-column albums list
+ * and a 2-columns photos list
+ */
+@media screen and (max-width: 480px) {
```

Comment:
Updated in https://github.com/cshold/jQuery-Facebook-Photo-Selector/pull/26/commits/9ad6795cd7f2676516f6a40328a92626fbb6d7df which also contains a fix for the Close button that was not correctly positioned (I have commited in the wrong branch the first time so I merged PhotosOfYou into Master)